core/clingutils/test/TClingUtilsTests.cxx

```diff
@@ -71,5 +71,11 @@ TEST(TClingUtilsTests, GetRealPath)
    std::remove("./symlink_realfile2");
    std::remove("./realfile1");
    std::remove("./realfile2");
+
+   // Test for mangling string
+   std::string mangled;
+   GetCppName(mangled, "some+input-string*with/special&characters123@abc$and spaces");
```

Comment:
Do we really use this interface for name mangling or here `mangled` is a misnomer?

Reply:
It's just replacing special characters in strings in such a way that it can be a valid C/C++ variable, I used mangling because I saw that in the function description, I'll rename it to something else.

Reply:
Phew, ok, maybe we will need to clarify that in the interface, too..